src/main/kotlin/no/fdk/fdk_public_service_harvester/adapter/FusekiAdapter.kt

```diff
@@ -16,15 +16,15 @@ private val LOGGER = LoggerFactory.getLogger(FusekiAdapter::class.java)
 @Service
 class FusekiAdapter(private val fusekiProperties: FusekiProperties) {
 
-    fun storeUnionModel(model: Model) =
-        try {
-            with(URL(fusekiProperties.unionGraphUri).openConnection() as HttpURLConnection) {
+    fun storeUnionModel(model: Model) {
+        with(URL(fusekiProperties.unionGraphUri).openConnection() as HttpURLConnection) {
+            try {
                 setRequestProperty("Content-type", "application/rdf+xml")
                 requestMethod = "PUT"
                 doOutput = true
 
                 OutputStreamWriter(outputStream).use {
-                    it.write(model.createRDFResponse(JenaType.RDF_XML))
+                    it.write(model.createRDFResponse(Lang.RDFXML))
```

Comment:
Hvorfor rdfxml og ikke turtle?

Reply:
Gamle fuseki leste feil for noen tripler når jeg brukte turtle, har ikke sjekket om det fortsatt er et problem. Men husker ikke hvilke tripler det var snakk om, blir kanskje vanskelig å sjekke :P
Et problem at vi bruker xml?